### ISSUE TITLE
LodeSTAR is not consistent across multiple trainings

### ISSUE THREAD

**Issue by HarshithBachimanchi** [DeepTrackAI/DeepTrack2#143]
Hi!

There seems to be a consistency issue with LodeSTAR. Given the same set of crops and training multiple independent LodeSTAR models, the training loss converges only in about 4 out of 10 cases. I've noticed this for multiple videos.

Have you noticed this? If yes, is there any solution for this problem?

**Comment by BenjaminMidtvedt** [comment]
I have noticed this in a few cases, yes. I have not quite figured out what makes it converge or not. But typically, the more you downsample the data, the more stable the training.

Also, the quality of the crops can matter. It can be a good idea to train a few epochs on fewer, high-quality crops to start with, before training on the full set of data.
